linux_os/guide/system/auditing/auditd_configure_rules/audit_dac_actions/audit_rules_dac_modification_fremovexattr/rule.yml

```diff
@@ -78,3 +78,4 @@ template:
     name: audit_rules_dac_modification
     vars:
         attr: fremovexattr
+        check_root_user: "true"
```

Comment:
The rule description needs to updated too.

Reply:
Yep. I'm still not convinced about the parameter name `check_root_user` though. I will put more thought into it.